lib/routes/callback/index.js

```diff
@@ -5,6 +5,44 @@ const { TooManyRequests } = require('http-errors')
 const { metricsInsert } = require('../metrics/query')
 const { registerUpdate } = require('./query')
 
+/**
+ * Allows users to request a callback from a Contact Tracer. If accepted, the request is added to the
+ * AWS callback queue with information about the user to call. The user's phone # is submitted as
+ * part of this request, but the phone # IS NOT stored with the user's database record. It is included
+ * in the body of the AWS callback queue message so that Contact Tracers know which number to call.
+ *
+ * Note that within the AWS callback queue the user's ID and phone # are stored together. This is a
+ * potential PII issue as it provides an identifiable mapping between users and their phone numbers.
+ *
+ * See schema.js/callback for details on what data can be submitted with this request.
+ *
+ * Users are identified by the 'id' in the claims of the request Authorization JWT header. The id value
+ * comes from the registration that occurred when the user installed the application.
+ *
+ * When the request comes in the user's registration record will be updated setting the last_callback
+ * column to the current timestamp IFF the last_callback column is empty. This prevents the same user
+ * from entering multiple callback requests.
+ *
+ * Users will be allowed to request additional callbacks IFF some other process clears the last_callback
```

Comment:
This is the second IFF I've seen. We should phrase correctly

Reply:
I agree. I'll change both to the full "if and only if" for clarity.